src/Orchard.Web/Modules/Orchard.Blogs/Migrations.cs

```diff
@@ -39,8 +39,8 @@ public class Migrations : DataMigrationImpl {
                     .WithPart("AutoroutePart", builder => builder
                         .WithSetting("AutorouteSettings.AllowCustomPattern", "True")
                         .WithSetting("AutorouteSettings.AutomaticAdjustmentOnEdit", "False")
-                        .WithSetting("AutorouteSettings.PatternDefinitions", "[{\"Name\":\"Title\",\"Pattern\":\"{Content.Slug}\",\"Description\":\"my-blog\"}]")
-                        .WithSetting("AutorouteSettings.DefaultPatternIndex", "0"))
+                        .WithSetting("AutorouteSettings.PatternDefinitions", "[{\"Name\":\"Title\",\"Pattern\":\"{Content.Slug}\",\"Description\":\"my-blog\",\"Culture\":\"\"}]")

```

Comment:
Remove all these default values in the migrations, and handle the case where there is now patterndefinition when you try to read it. At that time only create a default one, which will be saved if the user clicks Save in the UI. Or you actually CLONE the current ones for all the available cultures.